src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
```

Comment:
3. Javadoc: Consider adding Javadoc comments to document the purpose of the methods and any important information for future reference and maintenance.